Apps/Examples/Examples/All Examples/NavigationSimulatorExample.swift

```diff
@@ -41,7 +41,8 @@ final class NavigationSimulatorExample: UIViewController, ExampleProtocol {
     private func configureMap() {
         navigationSimulator = NavigationSimulator(viewport: mapView.viewport, route: sampleRouteLine)
 
-        mapView.location.options.puckType = .puck2D(.makeDefault(showBearing: true))
+        let configuration = Puck2DConfiguration(topImage: UIImage(named: "user_puck_icon"))
```

Comment:
It is important to have an image in this place. We have to unwrap it so in case of missing image we would have a crash with exact place in code describing the issue.
```suggestion
        let configuration = Puck2DConfiguration(topImage: UIImage(named: "user_puck_icon")!)
```